packages/user-operation-controller/package.json

```diff
@@ -71,10 +71,10 @@
   },
   "peerDependencies": {
     "@metamask/approval-controller": "^6.0.0",
-    "@metamask/gas-fee-controller": "^14.0.0",
+    "@metamask/gas-fee-controller": "^15.0.0",
```

Comment:
It looks like we forgot to update this controller. We should release this as well, otherwise we won't be able to update the GasFeeController or the TransactionController without introducing peer dependency warnings